lib/ansible/modules/files/stat.py

```diff
@@ -155,7 +155,7 @@
             description: Unix permissions of the file in octal
             returned: success, path exists and user can read stats
             type: octal
```

Comment:
Is `octal` actually a valid (return) type?

Reply:
I bet they mean a string with an octal number in it. So maybe we should return it as `mode` as well. Although I never planned to fix issues like this, but maybe I should too.

Reply:
I fixed it, and found another module that required fixing.